doc/changes/latest.rst

```diff
@@ -15,6 +15,7 @@ Fixes
 
 - Regressor names can now be invalid identifiers but will raise an error with :meth:`~glm.first_level.FirstLevelModel.compute_contrast` if combined to make an invalid expression (:gh:`3374` by `Yasmin Mzayek`_).
 - Fix :func:`~plotting.plot_connectome` which was raising a ``ValueError`` when ``vmax < 0`` (:gh:`3390` by `Paul Bogdan`_).
+- Fix display of colorbar in matrix plots which was overlapping in plt.subplots (:gh:`3403` by `Raphael Meudec`_).
```

Comment:
```suggestion
- Fix display of colorbar in matrix plots which was overlapping in :func:`~matplotlib.pyplot.subplots` (:gh:`3403` by `Raphael Meudec`_).
```

Hi @RaphaelMeudec, can this be a bit more specific or clear? Is the issue related to the function `plt.subplots` or rather how the nilearn `plot_matrix` colorbar code interacts with it?